specs/engage-digital_postman2.json

```diff
@@ -735,8 +735,8 @@
                     },
                     {
                       "key": "private",
-                      "value": "\u003cinteger\u003e",
-                      "description": "Created contents are public by default, set this parameter to \"1\" in order to create a private reply. It is NOT supported on every source.",
+                      "value": "\u003cboolean\u003e",
```

Comment:
as far as I can tell, this file is auto generated, the file to update is `specs/engage-digital_openapi3.yaml`. The schema type of this field should change from "integer" (already wrong) to "boolean".

Reply:
I changed my process to update `specs/engage-digital_openapi3.yaml` and then generate `specs/engage-digital_postman2.json` from it using `spectrum` (cf. https://github.com/ringcentral/engage-digital-api-docs/blob/master/specs/README.md#usage).
FTR the CI failed at the beginning because my version of `spectrum` was too recent. It worked after installing the 1.15.0 version using `go install github.com/grokify/spectrum@v1.15.0`.